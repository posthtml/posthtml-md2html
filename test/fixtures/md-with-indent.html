<div>
  <h2>Following, some markdown:</h2>
  <md>
    Lorem ipsum.
  </md>
</div>