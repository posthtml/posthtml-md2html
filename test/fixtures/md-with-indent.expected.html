<div>
  <h2>Following, some markdown:</h2>
  <p>  Lorem ipsum.</p>

</div>